run_wml_tests

```diff
@@ -170,8 +170,7 @@ class WesnothRunner:
                 path += "-debug"
         self.common_args = [path, "--nobanner"]
         if os.name == 'nt':
-            self.common_args.append("--wnoconsole")
-            self.common_args.append("--wnoredirect")
+            self.common_args.append("--no-log-to-file")
```

Comment:
Is there any reason why this is Windows-only? Does it have a different effect on Windows? It should be implicit on Mac and Linux due to the `-u` flag also being passed, which would mean adding it explicitly doesn't change anything.

Reply:
True, I forgot about that.

Reply:
Updated.